qst/src/main/java/io/deephaven/qst/array/Array.java

```diff
@@ -31,13 +31,6 @@
         return builder(type, data.length).add(data).build();
     }
 
-    static <T> Array<T> of(Type<T> type, Iterable<T> data) {
```

Comment:
I thought you were just removing the `Collection` ifs like:
```
        if (data instanceof Collection) {
            return of(type, (Collection<T>) data);
        }
 ```

Reply:
Okay. I can revert. I would like to have the check if we support iterable (I have seen it used as a semi-common pattern).